src/components/navbar.tsx

```diff
@@ -1,5 +1,96 @@
+import Image from "next/image";
+import Link from "next/link";
+
 export default function Navbar() {
+  const categoriesNav = [
+    "novidades",
+    "sapatos",
+    "botas",
+    "acessórios",
+    "promoções",
+    "brizza",
+  ];
+
+  const categoryBrizza = categoriesNav.pop();
+  const categoriesWithoutBrizza = categoriesNav.filter(
+    (category) => category !== "brizza",
+  );
```

Comment:
Achei legal como solucionou isso, ficou de maneira simples e clara. Porém, acredito que tu poderia trabalhar usando um last child do taiilwind para aplicar um estilo diferente no menu.

https://tailwindcss.com/docs/hover-focus-and-other-states#first-last-odd-and-even

Reply:
Ba que massa, não conhecia esse conceito do tailwind!